pulser-core/pulser/channels/dmm.py

```diff
@@ -31,16 +32,20 @@ class DMM(Channel):
     (of zero amplitude and phase). These Pulses are locally modulated by the
     weights of a `DetuningMap`, thus providing a local control over the
     detuning. The detuning of the pulses added to a DMM has to be negative,
-    between 0 and `bottom_detuning`. Channel targeting the transition between
-    the ground and rydberg states, thus encoding the 'ground-rydberg' basis.
+    between 0 and `bottom_detuning`, and the sum of the weights multiplied by
+    that detuning has to be blow `global_bottom_detuning`. Channel targeting
```

Comment:
I made this comment before your latest commit but perhaps the typo is still there?

Reply:
Oops sorry I did not see it :'(

Reply:
No no, I hadn't published it yet, I left it on standby. I made this remark because I noticed `global_bottom_detuning` and figured that could be confusing